guacamole/src/main/frontend/src/translations/en.json

```diff
@@ -905,6 +978,7 @@
     "SETTINGS_CONNECTIONS" : {
 
         "ACTION_ACKNOWLEDGE"          : "@:APP.ACTION_ACKNOWLEDGE",
+        "ACTION_IMPORT_CONNECTIONS"   : "Import",
```

Comment:
This should probably be declared generically as `ACTION_IMPORT` at the `APP` level, and then referenced here.